packages/presentation/lib/src/pages/widgets/game_form.dart

```diff
@@ -1,16 +1,21 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_bloc/flutter_bloc.dart';
-import 'package:presentation/src/bloc/events.dart';
-import 'package:presentation/src/bloc/game_bloc.dart';
-import 'package:presentation/src/bloc/state.dart';
+import 'package:presentation/src/game_bloc/bloc_tile.dart';
+import 'package:presentation/src/game_bloc/events.dart';
+import 'package:presentation/src/game_bloc/game_bloc.dart';
 
 const maxGuessNumber = 10;
 
 class GameForm extends StatelessWidget {
-  final GameState state;
+  final BlocTile tile;
+  final GameBloc bloc;
+
   static final _formKey = GlobalKey<FormState>();
 
-  const GameForm({Key? key, required this.state}) : super(key: key);
+  const GameForm({
+    Key? key,
+    required this.tile,
+    required this.bloc,
+  }) : super(key: key);
 
   String? _validator(String? text) => (text != null &&
           ((int.tryParse(text) ?? -1) > maxGuessNumber ||
```

Comment:
you can create validator file and move this logic there

Reply:
if I move it in bloc is it normal?

Reply:
so, I fixed this issue, but I merged get it here) sorry